src/ppx/ppx_eliom_utils.ml

```diff
@@ -285,6 +285,12 @@ module Context = struct
     | "client" | "client.start"
     | "eliom.client" | "eliom.client.start" -> `Client
     | _ -> invalid_arg "Eliom ppx: Not a context"
+  
+  let rpc_of_string = function
+    | "c_rpc"-> `Connected_rpc
+    | "c_rpc_o" -> `Connected_rpc_o
+    | "c_wrapper"-> `Connected_wrapper
```

Comment:
I would say :  
- rpc : wrapper
- cprc : connected rpc
- crpc_opt : optionnally connected rpc

what do you think @jrochel @ilankri @facelesspanda @vouillon @balat ?

Reply:
makes sense to me

Reply:
I am OK too with the suggested names.